utils/run-unasync.py

```diff
@@ -70,6 +70,7 @@ def main(check=False):
         "async_pull_request": "pull_request",
         "async_examples": "examples",
         "assert_awaited_once_with": "assert_called_once_with",
+        "pytest_asyncio": "pytest",
```

Comment:
Why is this needed now? Also, I think `@pytest_asyncio.fixture` and `@pytest.fixture` are probably clearer than `@fixture`, but that can be addressed further down the road.

Reply:
This is because pytest-asyncio's auto mode also takes care of reconfiguring any async fixtures declared with `@pytest.fixture` appropriately. In strict mode you have to use `@pytest_asyncio.fixture` instead.

I agree that `@pytest.fixture` is better than `@fixture` here, I'll change it.